tavern/printer.py

```diff
@@ -4,17 +4,19 @@
 
 
 def log_pass(test, response):
-    fmt = "PASSED: {:s} [{:d}]"
-    formatted = fmt.format(test["name"], response.status_code)
+    # pylint: disable=unused-argument
+    fmt = "PASSED: {:s}"
+    formatted = fmt.format(test["name"])
     logger.info(formatted)
 
 
 def log_fail(test, response, expected):
+    # pylint: disable=unused-argument
     # TODO print more info
     fmt = "FAILED: {:s} [{}]"
-    if response:
+    try:
         formatted = fmt.format(test["name"], response.status_code)
```

Comment:
Should doing this be handled by a method on the verifier or something? seems like it would be more flexible/extendable later?

Reply:
Definitely should be, the whole log_fail/log_pass should be reworked a lot

Reply:
specifically this https://github.com/taverntesting/tavern/issues/2